docs/tables/.ipynb_checkpoints/MagLoopAdvection-checkpoint.md

```diff
@@ -0,0 +1,3 @@
+## Brio-wu errors
```

Comment:
I think we don't need this file. Could you please remove it?

Reply:
done!